plugins/pulp_rpm/plugins/importers/yum/repomd/presto.py

```diff
@@ -36,7 +37,7 @@ def process_package_element(element):
         oldepoch=delta.attrib['oldepoch'],
         oldversion=delta.attrib['oldversion'],
         oldrelease=delta.attrib['oldrelease'],
-        filename=filename.text,
+        filename=os.path.join("drpms", os.path.basename(filename.text)),
```

Comment:
It's not required at all.

I just thought that it should be there. I thought we could ran into some data inconsistency if we would be syncing from repo with drpms in non standart place.